doc/source/drivers/raster/vrt.rst

```diff
@@ -856,8 +860,8 @@ GDAL provides a set of default pixel functions that can be used without writing
      - interpolate a value at time (or position) ``t`` given input sources beginning at ``t0`` with spacing ``dt`` using linear interpolation
    * - **inv**
      - 1
-     - -
-     - inverse (1./x). Note: no check is performed on zero division
+     - 1 (optional)
```

Comment:
The "Note: no check is performed on zero division" is outdated (0 is actually checked). Similar for the div case

Reply:
done